# English translations for PACKAGE package.
# Copyright (C) 2014 THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# Automatically generated, 2014.
#
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2014-04-11 22:18+0100\n"
"PO-Revision-Date: 2014-03-07 22:38+0000\n"
"Last-Translator: Automatically generated\n"
"Language-Team: none\n"
"Language: en_GB\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=ASCII\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: core/Rpg.py:53
msgid "ERROR_UNKNOWN_SELECTED_WORLD"
msgstr "Unknown selected world"

#: core/Rpg.py:60
msgid "ERROR_NO_SELECTED_PLAYER"
msgstr "A player is required"

#: core/Rpg.py:84
msgid "PLAYER_SELECTION"
msgstr "Player selection"

#: core/Rpg.py:85
msgid "CHOICE_NEW_PLAYER"
msgstr "Create a new player"

#: core/Rpg.py:86
msgid "CHOICE_EXISTING_PLAYER"
msgstr "Choose an existing player"

#: core/Rpg.py:89
msgid "CHOICE_QUESTION"
msgstr "Your choice? "

#: core/Rpg.py:96 core/command_factory.py:52 tests/commands/createPlayer.py:17
#: tests/commands/createPlayer.py:22 tests/commands/createPlayer.py:27
#: tests/commands/createPlayer.py:32 tests/commands/createPlayer.py:37
#: tests/commands/createPlayer.py:42 tests/commands/createPlayer.py:47
#: tests/commands/createPlayer.py:52 tests/commands/createPlayer.py:57
#: tests/commands/createPlayer.py:62
msgid "CREATE_PLAYER_COMMAND"
msgstr "create-player"

#: core/Rpg.py:110 core/commands/createPlayer.py:46
msgid "LOGIN_PROMPT"
msgstr "Login: "

#: core/Rpg.py:113 core/commands/createPlayer.py:54
msgid "PASSWORD_PROMPT"
msgstr "Password: "

#: core/Rpg.py:222
msgid "COMMAND_PROMPT"
msgstr "Command: "

#: core/command_factory.py:39 core/commands/help.py:38
#: tests/commands/help.py:26 tests/commands/help.py:61
#: tests/commands/look.py:15 tests/commands/look.py:34
msgid "LOOK_COMMAND"
msgstr "look"

#: core/command_factory.py:40 core/commands/help.py:22
#: tests/commands/help.py:18 tests/commands/help.py:45
#: tests/commands/talk.py:14 tests/commands/talk.py:19
#: tests/commands/talk.py:24 tests/commands/talk.py:29
#: tests/commands/talk.py:34 tests/commands/talk.py:39
#: tests/commands/talk.py:44 tests/commands/talk.py:49
#: tests/commands/talk.py:54
msgid "TALK_COMMAND"
msgstr "talk"

#: core/command_factory.py:41 core/commands/help.py:26
#: tests/commands/help.py:20 tests/commands/help.py:49
#: tests/commands/move.py:14 tests/commands/move.py:19
#: tests/commands/move.py:24 tests/commands/move.py:29
#: tests/commands/move.py:34 tests/commands/move.py:39
#: tests/commands/move.py:44 tests/commands/move.py:49
msgid "MOVE_COMMAND"
msgstr "move"

#: core/command_factory.py:42 core/commands/help.py:30
#: tests/commands/enter.py:14 tests/commands/enter.py:19
#: tests/commands/enter.py:24 tests/commands/enter.py:29
#: tests/commands/enter.py:34 tests/commands/enter.py:39
#: tests/commands/enter.py:44 tests/commands/enter.py:46
#: tests/commands/enter.py:51 tests/commands/enter.py:53
#: tests/commands/enter.py:58 tests/commands/enter.py:63
#: tests/commands/exit.py:53 tests/commands/exit.py:60
#: tests/commands/help.py:22 tests/commands/help.py:53
msgid "ENTER_COMMAND"
msgstr "enter"

#: core/command_factory.py:43 core/commands/help.py:34
#: tests/commands/exit.py:13 tests/commands/exit.py:18
#: tests/commands/exit.py:23 tests/commands/exit.py:28
#: tests/commands/exit.py:33 tests/commands/exit.py:38
#: tests/commands/exit.py:43 tests/commands/exit.py:48
#: tests/commands/exit.py:55 tests/commands/exit.py:62
#: tests/commands/help.py:24 tests/commands/help.py:57
msgid "EXIT_COMMAND"
msgstr "exit"

#: core/command_factory.py:44 core/commands/help.py:46
#: tests/commands/drop.py:54 tests/commands/drop.py:62
#: tests/commands/drop.py:69 tests/commands/drop.py:77
#: tests/commands/drop.py:84 tests/commands/drop.py:92
#: tests/commands/help.py:30 tests/commands/help.py:69
#: tests/commands/inventory.py:24 tests/commands/inventory.py:32
#: tests/commands/take.py:14 tests/commands/take.py:19
#: tests/commands/take.py:24 tests/commands/take.py:29
#: tests/commands/take.py:34 tests/commands/take.py:39
#: tests/commands/take.py:44 tests/commands/take.py:49
#: tests/commands/take.py:54 tests/commands/take.py:59
#: tests/commands/take.py:64 tests/commands/take.py:69
#: tests/commands/take.py:74 tests/commands/take.py:79
msgid "TAKE_COMMAND"
msgstr "take"

#: core/command_factory.py:45 core/commands/help.py:50
#: tests/commands/drop.py:14 tests/commands/drop.py:19
#: tests/commands/drop.py:24 tests/commands/drop.py:29
#: tests/commands/drop.py:34 tests/commands/drop.py:39
#: tests/commands/drop.py:44 tests/commands/drop.py:49
#: tests/commands/drop.py:56 tests/commands/drop.py:64
#: tests/commands/drop.py:71 tests/commands/drop.py:79
#: tests/commands/drop.py:86 tests/commands/drop.py:94
#: tests/commands/help.py:32 tests/commands/help.py:73
msgid "DROP_COMMAND"
msgstr "drop"

#: core/command_factory.py:46 core/commands/help.py:42
#: tests/commands/help.py:28 tests/commands/help.py:65
#: tests/commands/inventory.py:14 tests/commands/inventory.py:19
#: tests/commands/inventory.py:26 tests/commands/inventory.py:34
msgid "INVENTORY_COMMAND"
msgstr "inventory"

#: core/command_factory.py:47 core/commands/help.py:42
#: tests/commands/help.py:28 tests/commands/help.py:65
msgid "INVENTORY_SHORT_COMMAND"
msgstr "inv"

#: core/command_factory.py:48 core/commands/help.py:54
#: tests/commands/help.py:15 tests/commands/help.py:34
#: tests/commands/help.py:41 tests/commands/help.py:77
msgid "HELP_COMMAND"
msgstr "help"

#: core/command_factory.py:72 core/commands/help.py:58
#: tests/commands/help.py:36 tests/commands/help.py:81
msgid "QUIT_COMMAND"
msgstr "quit"

#: core/command_factory.py:72 core/commands/help.py:58
#: tests/commands/help.py:36 tests/commands/help.py:81
msgid "QUIT_SHORT_COMMAND"
msgstr "q"

#: core/command_factory.py:81 core/command_factory.py:94
msgid "ERROR_UNKNOWN_COMMAND"
msgstr "Unknown command"

#: core/commands/createPlayer.py:23 tests/commands/createPlayer.py:19
#: tests/commands/createPlayer.py:24
msgid "ERROR_SIGNUP_NOT_ENOUGH_ARGUMENTS"
msgstr "More arguments are required to create a player"

#: core/commands/createPlayer.py:28 core/commands/createPlayer.py:49
#: tests/commands/createPlayer.py:29 tests/commands/createPlayer.py:34
msgid "ERROR_SIGNUP_LOGIN_ALREADY_USED"
msgstr "This login is already used"

#: core/commands/createPlayer.py:32 tests/commands/createPlayer.py:39
#: tests/commands/createPlayer.py:44
msgid "ERROR_SIGNUP_INVALID_GENDER"
msgstr "Invalid gender"

#: core/commands/createPlayer.py:36 tests/commands/createPlayer.py:49
#: tests/commands/createPlayer.py:54
msgid "ERROR_SIGNUP_INVALID_SPECIES"
msgstr "Invalid species"

#: core/commands/createPlayer.py:55
msgid "CONFIRM_PASSWORD_PROMPT"
msgstr "Password confirmation: "

#: core/commands/createPlayer.py:58
msgid "ERROR_SIGNUP_PASSWORDS"
msgstr "The password do not match"

#: core/commands/createPlayer.py:64
msgid "GENDER_SELECTION"
msgstr "Gender selection"

#: core/commands/createPlayer.py:70
msgid "GENDER_PROMPT"
msgstr "Gender: "

#: core/commands/createPlayer.py:84
msgid "SPECIES_SELECTION"
msgstr "Species selection"

#: core/commands/createPlayer.py:91
msgid "SPECIES_PROMPT"
msgstr "Species: "

#: core/commands/createPlayer.py:104 tests/commands/createPlayer.py:59
msgid "PLAYER_CREATION_CONFIRMATION"
msgstr "Player created"

#: core/commands/drop.py:17 tests/commands/drop.py:16
#: tests/commands/drop.py:21
msgid "ERROR_DROP_NO_ITEM_GIVEN"
msgstr "What shall I drop?"

#: core/commands/drop.py:27 tests/commands/drop.py:46
#: tests/commands/drop.py:51
msgid "ERROR_DROP_INVALID_QUANTITY"
msgstr "Invalid quantity"

#: core/commands/drop.py:34 tests/commands/drop.py:26
#: tests/commands/drop.py:31
msgid "ERROR_DROP_UNKNOWN_ITEM"
msgstr "I have none of those"

#: core/commands/drop.py:39 tests/commands/drop.py:36
#: tests/commands/drop.py:41
msgid "ERROR_DROP_ITEM_NOT_AVAILABLE"
msgstr "I have none of those"

#: core/commands/drop.py:41 tests/commands/drop.py:58
#: tests/commands/drop.py:66
#, python-format
msgid "ERROR_DROP_QUANTITY_TOO_HIGH_%s"
msgstr "I don't have enough %s to drop"

#: core/commands/drop.py:50 tests/commands/drop.py:73
#: tests/commands/drop.py:88
#, python-format
msgid "DROP_CONFIRMATION_%(quantity)s_%(name)s"
msgstr "You drop %(quantity)s %(name)s"

#: core/commands/enter.py:21 tests/commands/enter.py:16
#: tests/commands/enter.py:21
msgid "ERROR_ENTER_NO_PLACE_GIVEN"
msgstr "I can't enter into nothing"

#: core/commands/enter.py:27 core/commands/exit.py:27
#: tests/commands/enter.py:26 tests/commands/enter.py:31
#: tests/commands/enter.py:48 tests/commands/enter.py:55
#: tests/commands/exit.py:25 tests/commands/exit.py:30
#: tests/commands/exit.py:45 tests/commands/exit.py:50
msgid "ERROR_ENTER_PLACE_NOT_AVAILABLE"
msgstr "There is no such place here"

#: core/commands/enter.py:33 tests/commands/enter.py:60
#: tests/commands/enter.py:65
msgid "ENTER_CONFIRMATION"
msgstr "You enter."

#: core/commands/exit.py:21 tests/commands/exit.py:15
#: tests/commands/exit.py:20
msgid "ERROR_EXIT_NO_PLACE_GIVEN"
msgstr "I can't exit out of nothing"

#: core/commands/exit.py:31 tests/commands/exit.py:57
#: tests/commands/exit.py:64
msgid "EXIT_CONFIRMATION"
msgstr "You are now outside"

#: core/commands/help.py:22 tests/commands/help.py:18
#: tests/commands/help.py:45
msgid "CHARACTER_TOKEN"
msgstr "character name"

#: core/commands/help.py:22 tests/commands/help.py:18
#: tests/commands/help.py:45
msgid "SENTENCE_TOKEN"
msgstr "sentence"

#: core/commands/help.py:23 tests/commands/help.py:19
#: tests/commands/help.py:46
msgid "TALK_COMMAND_DESCRIPTION"
msgstr "Talk to a character"

#: core/commands/help.py:27 tests/commands/help.py:21
#: tests/commands/help.py:50
msgid "MOVE_COMMAND_DESCRIPTION"
msgstr "Go to the indicated direction"

#: core/commands/help.py:31 tests/commands/help.py:23
#: tests/commands/help.py:54
msgid "ENTER_COMMAND_DESCRIPTION"
msgstr ""
"Enter in the selected place (if the place is available in the current cell)"

#: core/commands/help.py:35 tests/commands/help.py:25
#: tests/commands/help.py:58
msgid "EXIT_COMMAND_DESCRIPTION"
msgstr "Exit from the selected place (if the current cell is the place exit)"

#: core/commands/help.py:39 tests/commands/help.py:27
#: tests/commands/help.py:62
msgid "LOOK_COMMAND_DESCRIPTION"
msgstr ""
"List everything which is in the current area (characters, items, neighbour "
"areas...)"

#: core/commands/help.py:43 tests/commands/help.py:29
#: tests/commands/help.py:66
msgid "INVENTORY_COMMAND_DESCRIPTION"
msgstr "List the items the player has in his inventory"

#: core/commands/help.py:46 core/commands/help.py:50 tests/commands/help.py:30
#: tests/commands/help.py:32 tests/commands/help.py:69
#: tests/commands/help.py:73
msgid "ITEM_NAME_TOKEN"
msgstr "item name"

#: core/commands/help.py:46 core/commands/help.py:50 tests/commands/help.py:30
#: tests/commands/help.py:32 tests/commands/help.py:69
#: tests/commands/help.py:73
msgid "QUANTITY_TOKEN"
msgstr "quantity"

#: core/commands/help.py:47 tests/commands/help.py:31
#: tests/commands/help.py:70
msgid "TAKE_COMMAND_DESCRIPTION"
msgstr "Take some items on the ground"

#: core/commands/help.py:51 tests/commands/help.py:33
#: tests/commands/help.py:74
msgid "DROP_COMMAND_DESCRIPTION"
msgstr "Drop some items from your inventory"

#: core/commands/help.py:55 tests/commands/help.py:35
#: tests/commands/help.py:78
msgid "HELP_COMMAND_DESCRIPTION"
msgstr "Display the help menu"

#: core/commands/help.py:59 tests/commands/help.py:37
#: tests/commands/help.py:82
msgid "QUIT_COMMAND_DESCRIPTION"
msgstr "Quit the game"

#: core/commands/help.py:64 tests/commands/help.py:17
msgid "AVAILABLE_COMMANDS_TITLE"
msgstr "Available commands"

#: core/commands/look.py:63 tests/commands/look.py:18
#, python-format
msgid "CURRENT_REGION_%s"
msgstr "You are in %s"

#: core/commands/look.py:67 tests/commands/look.py:20
msgid "PRESENT_CHARACTERS"
msgstr "You see these characters arround:"

#: core/commands/look.py:73 tests/commands/look.py:23
msgid "AVAILABLE_DIRECTIONS"
msgstr "You can go in the following directions:"

#: core/commands/look.py:79 tests/commands/look.py:26
msgid "AVAILABLE_PLACES"
msgstr "You see the following places:"

#: core/commands/look.py:85 tests/commands/look.py:29
msgid "AVAILABLE_ITEMS"
msgstr "You see the following items"

#: core/commands/move.py:19 tests/commands/move.py:16
#: tests/commands/move.py:21
msgid "ERROR_MOVE_NO_DIRECTION_GIVEN"
msgstr "Where shall I go?"

#: core/commands/move.py:23 tests/commands/move.py:26
#: tests/commands/move.py:31
#, python-format
msgid "ERROR_MOVE_INVALID_DIRECTION_%s"
msgstr "%s is not a valid direction"

#: core/commands/move.py:30 tests/commands/move.py:36
#: tests/commands/move.py:41
msgid "ERROR_MOVE_DIRECTION_NOT_AVAILABLE"
msgstr "I can't go there"

#: core/commands/move.py:37 tests/commands/move.py:46
#, python-format
msgid "MOVE_CONFIRMATION_%s"
msgstr "lets go %s"

#: core/commands/take.py:11 tests/commands/take.py:16
#: tests/commands/take.py:21
msgid "ERROR_TAKE_NO_ITEM_GIVEN"
msgstr "What shall I take?"

#: core/commands/take.py:20 tests/commands/take.py:46
#: tests/commands/take.py:51
msgid "ERROR_TAKE_INVALID_QUANTITY"
msgstr "Invalid quantity"

#: core/commands/take.py:27 tests/commands/take.py:26
#: tests/commands/take.py:31
msgid "ERROR_TAKE_UNKNOWN_ITEM"
msgstr "I don't see this here."

#: core/commands/take.py:34 tests/commands/take.py:36
#: tests/commands/take.py:41
msgid "ERROR_TAKE_ITEM_NOT_AVAILABLE"
msgstr "I don't see this here."

#: core/commands/take.py:37 tests/commands/take.py:56
#: tests/commands/take.py:61
msgid "ERROR_TAKE_QUANTITY_TOO_HIGH"
msgstr "There are not enough items of this kind"

#: core/commands/take.py:46 tests/commands/take.py:66
#: tests/commands/take.py:76
#, python-format
msgid "TAKE_CONFIRMATION_%(quantity)s_%(name)s"
msgstr "You take %(quantity)s %(name)s"

#: core/commands/talk.py:18 tests/commands/talk.py:16
#: tests/commands/talk.py:21
msgid "ERROR_TALK_NO_CHARACTER_GIVEN"
msgstr "Who must I talk to?"

#: core/commands/talk.py:20 tests/commands/talk.py:26
#: tests/commands/talk.py:31
msgid "ERROR_TALK_NO_SENTENCE_GIVEN"
msgstr "What must I say?"

#: core/commands/talk.py:29 tests/commands/talk.py:36
#: tests/commands/talk.py:41
msgid "ERROR_TALK_UNKNOWN_CHARACTER"
msgstr "Unknow character"

#: core/commands/talk.py:36 tests/commands/talk.py:46
msgid "ERROR_TALK_UNKNOWN_SENTENCE"
msgstr "What?"

#: models/area.py:20 tests/commands/look.py:24 tests/commands/look.py:37
#: tests/commands/move.py:44 tests/commands/move.py:46
#: tests/commands/move.py:49 tests/commands/move.py:51
msgid "DIRECTION_KEY_NORTH"
msgstr "north"

#: models/area.py:21 tests/commands/move.py:34 tests/commands/move.py:39
msgid "DIRECTION_KEY_EAST"
msgstr "east"

#: models/area.py:22
msgid "DIRECTION_KEY_SOUTH"
msgstr "south"

#: models/area.py:23
msgid "DIRECTION_KEY_WEST"
msgstr "west"

#: models/area.py:38
msgid "AREA_TYPE_DUNGEON"
msgstr "dungeon"

#: models/area.py:38
msgid "AREA_TYPE_LAND"
msgstr "land"

#: models/place.py:15 models/place.py:29 models/place.py:39 models/place.py:49
#: tests/commands/enter.py:24 tests/commands/exit.py:23
#: tests/commands/exit.py:28
msgid "PLACE_TYPE_CAVE"
msgstr "cave"

#: models/place.py:15 models/place.py:27 models/place.py:37 models/place.py:47
#: tests/commands/enter.py:44 tests/commands/enter.py:46
#: tests/commands/enter.py:51 tests/commands/enter.py:53
#: tests/commands/enter.py:58 tests/commands/enter.py:63
#: tests/commands/exit.py:43 tests/commands/exit.py:48
#: tests/commands/exit.py:53 tests/commands/exit.py:55
#: tests/commands/exit.py:60 tests/commands/exit.py:62
msgid "PLACE_TYPE_DUNGEON"
msgstr "dungeon"

#: models/place.py:25 models/place.py:35 models/place.py:45
#: tests/commands/enter.py:36 tests/commands/enter.py:41
#: tests/commands/exit.py:35 tests/commands/exit.py:40
msgid "ERROR_UNKNOWN_PLACE_TYPE"
msgstr "Unknown place type"

#: models/place.py:96
msgid "ERROR_PLACE_GENERATION"
msgstr "An error occured during the place loading"

#: models/player.py:33
msgid "ERROR_CONNECT_INVALID_CREDENTIALS"
msgstr "Invalid login or password"

#~ msgid "GENERATING_WAITING_TEXT"
#~ msgstr "Loading place..."
